test/editor/operations-test.js

```diff
@@ -44,6 +46,23 @@ describe('editor/operations: formatting operations', () => {
             expect(model.serializeParts()).toEqual([{ "text": "hello _world_!", "type": "plain" }]);
         });
 
+        it('works for escaping backticks in between texts', () => {
+            const renderer = createRenderer();
+            const pc = createPartCreator();
+            const model = new EditorModel([
+                pc.plain("hello ` world!"),
+            ], pc, renderer);
+
+            const range = model.startRange(model.positionForOffset(0, false),
+                model.positionForOffset(13, false));  // hello ` world
+
+            expect(range.parts[0].text.trim().includes("`")).toBeTruthy();
+            expect(longestBacktickSequence(range.parts[0].text.trim())).toBe(1);
+            expect(model.serializeParts()).toEqual([{ "text": "hello ` world!", "type": "plain" }]);
+            formatRangeAsCode(range);
+            expect(model.serializeParts()).toEqual([{ "text": "``hello ` world``!", "type": "plain" }]);
+        });
```

Comment:
You should test that it works for longer sequences and that toggling it also works

Reply:
> You should test that it works for longer sequences and that toggling it also works

I didn't wanted to write too much tests, it actually works for all the case you stated even for longer sequence, i attached a video up there, you probably missed it. I can drop new video for you to see

Reply:
There is no such thing as "too much tests" - tests protect against this feature being broken by accident by contributions later down the line, the video only tests it working at the time of this PR being made. Please write tests which exercise all edges. Its great work, just needs testing so it doesn't accidentally regress through the mistakes of others.

Reply:
> There is no such thing as "too much tests" - tests protect against this feature being broken by accident by contributions later down the line, the video only tests it working at the time of this PR being made. Please write tests which exercise all edges. Its great work, just needs testing so it doesn't accidentally regress through the mistakes of others.

Ok thanks but Can you pls be clear about the longer sequence you meant like give an example so i can write proper test for it.  if you mean escaping longer backticks in between text like hello```world, then i can just write test for that.

Reply:
I think i now understand you, I have written more tests with all the edge cases i could think of, i'll appreciate if there is more you think i should add

> > There is no such thing as "too much tests" - tests protect against this feature being broken by accident by contributions later down the line, the video only tests it working at the time of this PR being made. Please write tests which exercise all edges. Its great work, just needs testing so it doesn't accidentally regress through the mistakes of others.
> 
> Ok thanks but Can you pls be clear about the longer sequence you meant like give an example so i can write proper test for it. if you mean escaping longer backticks in between text like hello```world, then i can just write test for that.

I think i now understand you, I have written more tests with all the edge cases i could think of, i'll appreciate if there is more you think i should add